e2e/podman/input/registry-auths.hcl

```diff
@@ -0,0 +1,120 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# This job runs after the private registry is up and running, when we know
+# address and port provided by the bridge network. It is a sysbatch job
+# that writes these files on every linux client.
+#   - /usr/local/bin/docker-credential-test.sh
```

Comment:
this is "docker-credential-" rather than "podman-credential-" because the registry itself is docker?

Reply:
It's just because the "official" credentials helpers in the ecosystem are provided by docker and their documentation prefixes everything using "docker-credential-"

https://github.com/docker/docker-credential-helpers/tree/master

I suppose we could search for either prefix, but unless someone actually asks for it I think we should just keep it simple and working with the known tools.